hiplot_mlflow.py

```diff
@@ -0,0 +1,166 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import uuid
+from typing import Union
+from urllib.parse import parse_qs, urlparse
+
+import hiplot
+from mlflow import search_runs, tracking
+from numpy import finfo, float64
+from pandas import DataFrame
+
+_positive_inf = finfo(float64).max
+_negative_inf = finfo(float64).min
+
+
+def fetch(name: str, include_tags: bool = False) -> hiplot.Experiment:
+    """Fetch experiments from the MLflow experiments tracking,
+    by experiment name.
+
+    Parameters
+    ----------
+    name: str
+        The name of the tracked experiment to look up
+    include_tags: bool
+        Whether to include tags as well in the resulting HiPlot experiment (False)
+
+    Returns
+    -------
+    hiplot.Experiment
+        The resulting experiment
+    """
+    return fetch_by_id(_get_experiment_id_from_name(name), include_tags)
+
+
+def fetch_by_id(id: Union[int, str], include_tags: bool = False) -> hiplot.Experiment:
+    """Fetch experiments from the MLflow experiments tracking,
+    by experiment ID.
+
+    Parameters
+    ----------
+    id: int|str
+        The ID if the experiment to look up
+    include_tags: bool
+        Whether to include tags as well in the resulting HiPlot experiment (False)
+
+    Returns
+    -------
+    hiplot.Experiment
+        The resulting experiment
+    """
+    df = search_runs([id])
+    return _create_experiment_from_dataframe(df, include_tags)
+
+
+def fetch_by_uri(uri: str) -> hiplot.Experiment:
+    """Fetch experiments from the MLflow experiments tracking,
+    by full URI.
+
+    The URI can take these forms:
+    * `mlflow://id/<id>` to look up experiments by ID
+    * `mlflow://name/<name>` to look up experiments by name
+
+    You can also add a query string to modify the results:
+    * Setting `tags` to any value will return results with
+      tags included. For example: `mlflow://id/<id>?tags=yes`
+
+    Parameters
+    ----------
+    uri: str
+        The fetcher URI to use
+
+    Returns
+    -------
+    hiplot.Experiment
+        The resulting experiment
+    """
+    # Only apply this fetcher if the URI matches our schema
+    parsed_uri = urlparse(uri)
+    if parsed_uri.scheme != "mlflow" or parsed_uri.netloc not in {"name", "id"}:
+        # It's not what we can handle
+        raise hiplot.ExperimentFetcherDoesntApply()
+
+    reference_type = parsed_uri.netloc
+    reference = parsed_uri.path.strip("/")
+    include_tags = parse_qs(parsed_uri.query).get("tags") is not None
+    if reference_type == "id":
+        return fetch_by_id(reference, include_tags)
+    else:
+        return fetch(reference, include_tags)
+
+
+def _get_experiment_id_from_name(name: str) -> int:
+    """Get an MLflow experiment's ID from its name
+    """
+    client = tracking.MlflowClient()
+    experiment = client.get_experiment_by_name(name)
+    if not experiment:
```

Comment:
I think this is more idiomatic than relying on the 'truthiness' of `None` in this kind of situation. (More generally: I am personally not a fan of the use of truthiness of complex objects in `if` statements like this)

```suggestion
    if experiment is not None:
```